lit_tests/kernel/wave/expansion.py

```diff
@@ -237,6 +238,7 @@ def test_gemm():
         graph = gemm()
         IndexingContext.current().finalize()
         expand_graph(graph, constraints)
+        visualize_graph(graph.get_subgraph("region_0"), "gemm.png")
```

Comment:
Do we want to keep it in unrelated test?

Reply:
Sure, I can move this out to its own visualization test.